docs/en/inference/convert_tutorial.md

```diff
@@ -1,17 +1,32 @@
 ## Inference - Model Conversion Tutorial
 
+MindOCR Inference support infer from MindOCR trained models and third-party(PaddelOCR and mmOCR) models.
```

Comment:
MindOCR supports model inference for models trained using MindOCR and models converted from third-party toolboxes (PaddelOCR and MMOCR).